.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -49,6 +49,20 @@ accessibility is impacted and tested. For more info, check out the
       have not been included_
 - [ ] I need help with writing tests
 
+## Does this change need to be communicated?
```

Comment:
Given this template is so large already and this feels very similar to `## Added to documentation?` I think we should look at combining them into a single section.

Reply:
That's a good point! I've updated this -- latest commit rolls the documentation section into the new one that I have added :)